test/e2e/performanceprofile/functests/2_performance_update/memorymanager.go

```diff
@@ -50,8 +51,9 @@ var _ = Describe("[rfe_id: 43186][memorymanager] Memorymanager feature", Label(s
 	var (
 		workerRTNodes           []corev1.Node
 		profile, initialProfile *performancev2.PerformanceProfile
-		performanceMCP          string
+		resourcePool            string
```

Comment:
`resourcePool` is not clear IMO.
 `poolName`  maybe? 
Pool can be `NodePool` or `MachineConfigPool`

Reply:
fixed in the latest commit